src/NServiceBus.RavenDB/Internal/NoOpLogManager.cs

```diff
@@ -1,8 +1,8 @@
 namespace NServiceBus.RavenDB
 {
-using System;
-using Raven.Abstractions.Extensions;
-using Raven.Abstractions.Logging;
+    using System;
```

Comment:
This looks vierd, is this a tabs vs space thing or just some new formatting in the GH ui?